src/exercise/01.js

```diff
@@ -1,29 +1,29 @@
 // useState: greeting
 // http://localhost:3000/isolated/exercise/01.js
 
-import * as React from 'react'
+import React, { useState } from 'react'
```

Comment:
Nice, this is how we're doing it on our projects typically. Makes the subsequent code much easier to read.